src/components/molecules/header/Header.styles.jsx

```diff
@@ -0,0 +1,50 @@
+import styled from "styled-components";
+import { zindex, spaces, contentW, sizes, colors } from "@utils";
+
+export const HeaderWrapper = styled.header`
+  position: fixed;
+  top: 0;
+  z-index: ${zindex.header};
+  width: 100%;
+  height: ${spaces.ssm};
+  display: flex;
+`;
+
+export const HeaderContainer = styled.div`
+  position: relative;
+  max-width: ${contentW.lg};
+  width: 100%;
+  margin: 0 auto;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
```

Comment:
There is something wrong in your responsive after `1500px` 
![Screenshot from 2024-06-07 17-04-42](https://github.com/Isynopoulos1/wedding/assets/30130006/aea5e8e0-666e-48c4-948b-6e1a923834fc)

As you can see there is no space between the logo and the screen. That sound incorrect. Same for the otherside

Reply:
done